Formula/proofable.rb

```diff
@@ -0,0 +1,20 @@
+class Proofable < Formula
+  desc "Certifying any digital assets (files) to public blockchains"
+  homepage "https://docs.proofable.io"
+  url "https://github.com/SouthbankSoftware/proofable/archive/v0.2.2.tar.gz"
+  sha256 "11e4e76cb2998168e047436fb9ef414b152e1513c80f1ee0d6093da815810c5e"
+  license "AGPL-3.0"
+  head "https://github.com/SouthbankSoftware/proofable.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["APP_VERSION"] = version
+    system "make", "build"
+    bin.install "proofable-cli"
+  end
+
+  test do
+    assert_equal "proofable-cli version #{version}", shell_output("#{bin}/proofable-cli --version").strip
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
@SMillerDev thanks for the tip. I have fixed the test issue in the commit f4ff447. However, I receive the error `GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)` from the CI. Is this a hard requirement?